install-dev/fixtures/fashion/data/order_detail.xml

```diff
@@ -53,7 +53,7 @@
                       product_ean13="" product_upc="" product_reference="demo_2" product_supplier_reference=""
                       product_weight="0.000000" tax_computation_method="0" tax_name="" tax_rate="0.000"
                       ecotax="0.000000" ecotax_tax_rate="0.000" discount_quantity_applied="0" download_nb="0"
-                      download_deadline="0000-00-00 00:00:00" total_price_tax_incl="27.000000"
+                      download_deadline="0000-00-00 00:00:00" total_price_tax_incl="23.900000"
                       total_price_tax_excl="23.900000" unit_price_tax_incl="23.900000" unit_price_tax_excl="23.900000"
```

Comment:
Why do you need to change the tax included price on a fixture? The bug came from the fixture itself?

Reply:
Yes, this was the only field where tax included was different from tax excluded Which is why we didn't understand why this bug only happened on this specific order We could ignore it but I think it's better to fix it
Although the real best fix would be to use real tax included values but I think this will be done some other time

>